ingest/src/test/scala/hydra/ingest/http/IngestionEndpointSpec.scala

```diff
@@ -41,6 +45,8 @@ final class IngestionEndpointSpec
   implicit val guavaCache: Cache[SchemaWrapper] = GuavaCache[SchemaWrapper]
 
   private val noAuth = new AccessControlService[IO](mock[AwsSecurityService[IO]], AwsConfig(None, isAwsIamSecurityEnabled = false))
+  private val testSubject = Subject.createValidated("dvs.test.v0.Testing").get
+  private val timestampValidationCutoffDate: Instant = InstantUtils.dateStringToInstant("20230711")
```

Comment:
The same. Just use date. We need converter from string only for Decoder

Reply:
Done.